Friendly-plans/app/src/main/java/pg/autyzm/friendly_plans/view/task_create/TaskCreateFragment.java

```diff
@@ -294,7 +294,7 @@ private void showToastMessage(int resourceStringId) {
 
     @Override
     public void eventListStep(View view) {
-        Long taskId = saveOrUpdate();
+        taskId = saveOrUpdate();
```

Comment:
Nice work 👍 Thanks. Could you please add a test that proves it is working? I think it could be an additional assert in UI test. If you find any problems please let me know.

Reply:
I added whenMovingBackFromStepListNewTaskCanBeUpdated test in TaskCreateActivityTest